content/management/configuration-api/v3.3/index.mdx

```diff
@@ -1794,8 +1799,10 @@ Registers a new private property for a given **Client Id**.
 | `range` __*__              | No                  | `object`   | Range of values that properties can be set to                                         |
 | `range.from`               | No                  | `int`      | Only values **equal** or **greater** than this parameter can be set to this property  |
 | `range.to`                 | No                  | `int`      | Only values **equal** or **lower** than this parameter can be set to this property    |
+| `default_value` __**__     | No                  | `type`     | Default value of property; validated by **domain** or **range**, if one exists        |
 
 __*)__ Only one `domain` and one `range` can be set for a single property.
+__**)__ Register is possible only if `access.<location>` set to `license` or `group`.
```

Comment:
```suggestion
__**)__ Registering a property is possible only if `access.<location>` is set to `license` or to `group`.
```

Reply:
So if you want to access a property from chat you need to register it for license/group + chat?

Reply:
registering chat / thread / event property is not possible with default value

Reply:
ok, ok, I forgot it's only for a property with default value :D

Reply:
yes, my description was little bit wrong but i fixed it now should be more clear :)